www/js/app.js

```diff
@@ -836,6 +869,12 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
      * @param {String} htmlArticle
      */
     function displayArticleContentInIframe(dirEntry, htmlArticle) {
+        // Display Bootstrap warning alert if the landing page contains active content
+        var containsActiveContent = regexpActiveContent.test(htmlArticle);
```

Comment:
It would be better to move this test, so that it is executed only if `!params.hideActiveContentWarning && params.isLandingPage`
It would save some useless CPU cycles

Reply:
@mossroy OK, thanks, will do!

Reply:
Commit https://github.com/kiwix/kiwix-js/pull/467/commits/778bc0a6a565775f98a441621a5a499ca1d740b1 addresses this comment.

Reply:
Looks good